dev/App.js

```diff
@@ -14,7 +14,7 @@ import 'brace/theme/textmate';
 // https://github.com/plotly/react-chart-editor#mapbox-access-tokens
 import ACCESS_TOKENS from '../accessTokens';
 
-import {customConfigTest} from '../src/__stories__';
+// import {customConfigTest} from '../src/__stories__';
```

Comment:
I put it in `../src/__stories`, because when i had it in `dev/App.js` the imports that file has were causing issues with the percy tests. I guess this could go into it's separate file. But yes we can just comment out. Not exactly sure though what's causing the issue.. maybe that we reimport plotly.js in that storybook file, from a different place..